noprint.js

```diff
@@ -10,7 +10,7 @@ if (noCopy)
 	    return false;
 	}
  	document.onkeydown = function() {
-		if(event.ctrlKey == true && event.keyCode == 83) {
+		if((event.ctrlKey == true || event.metaKey == true) && event.keyCode == 83) {

```

Comment:
keyCode is deprecated, you might want to change this to `code` instead

Reply:
Thanks for the suggestion. Will add that ASAP.